LanguageFeatures/Augmentation-libraries/augmenting_declarations_A05_t02.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Because of augmentations, non-abstract class, mixin, mixin class,
+/// enum, extension type, and extension declarations are now allowed to contain
+/// abstract member declarations, as long as those members are equipped with a
+/// body by an augmentation declaration.
+///
+/// @description Checks that it is not an error if a body of an abstract static
+/// member is provided by an augmentation.
```

Comment:
Cool!

This shows that we need to think about "abstract static" members. I raised the question here: https://github.com/dart-lang/language/issues/4008.

So let's move the "abstract static" declarations out of this PR and into a separate one, and delay that other PR until https://github.com/dart-lang/language/issues/4008 has been resolved.